admin-ui/src/components/settings/settings-navigation.tsx

```
@@ -0,0 +1,66 @@
+import { AnimatedGradientText } from "@/components/ui/animated-gradient-text"
+import { cn } from "@/lib/utils"
+import type { SettingsSection } from "@/hooks/use-active-section"
```

ESLint

[error] Expected "@/hooks/use-active-section" (type-internal) to come before "@/lib/utils" (value-internal).

(perfectionist/sort-imports)